docs/openapi/swagger.yml

```diff
@@ -0,0 +1,392 @@
+openapi: 3.0.3
+info:
+  title: Pessimism REST API
+  description: >-
+    This document specifies the REST API for the Pessimism application software. <br><br>
+    **NOTE - This spec is still WIP and will be updated/changed as the application evolves.
+    It's also worth noting that this spec is not a complete representation of the application's current functionality.**
+  license:
+    name: MIT License
+    url: https://github.com/base-org/pessimism/blob/master/LICENSE
+  version: 0.0.1
+externalDocs:
+  description: Find out more about Pessimism's architecture
+  url: 'https://github.com/base-org/pessimism/docs/ARCHITECTURE.md'
+servers:
+  - url: 'http://127.0.0.1:8080'
```

Comment:
just a thought, maybe worth putting in a placeholder external than localhost ?

Reply:
Fixed!